### PR TITLE
Add container images

### PR DESCRIPTION
**Pull request by discordianfish**

### CODE REVIEW

File: images/stable-baselines/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+ARG on
+FROM python:${on}
```
Comment (alexpalms) [inline images/stable-baselines/Dockerfile:2]: SB works only with max python 3.7 (due to tensorflow limitations)

File: images/stable-baselines/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN apt-get -qy update && \
+    apt-get -qy install libgl1 && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get -qy clean
```
Comment (alexpalms) [inline images/stable-baselines/Dockerfile:7]: There are other libs that need to be installed to use SB: https://stable-baselines.readthedocs.io/en/master/guide/install.html

File: images/stable-baselines/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
+
+# Copy arena to tmp since bind-mount is read-only and pip doesn't support
+# out-of-tree builds.
+RUN --mount=target=/usr/src/arena,type=bind,source=. \
+    cp -r /usr/src/arena /tmp/arena && \
+    pip install /tmp/arena[stable-baselines] && \
```
Comment (alexpalms) [inline images/stable-baselines/Dockerfile:15]: in addition to this, one needs to install the proper TF version, either CPU or GPU. Do we want to add it here?

File: images/stable-baselines/Dockerfile (hunk 2 above)
Comment (discordianfish) [reply images/stable-baselines/Dockerfile:7]: It's a bit unclear to me why these would be needed. They are all build time dependencies..

File: images/stable-baselines/Dockerfile (hunk 3 above)
Comment (discordianfish) [reply images/stable-baselines/Dockerfile:15]: Good point. Do you know what is additionally need for CPU and GPU?

File: images/stable-baselines/Dockerfile (hunk 1 above)
Comment (discordianfish) [reply images/stable-baselines/Dockerfile:2]: Ok, I've updated it to use 3.7

File: images/stable-baselines/Dockerfile (hunk 2 above)
Comment (alexpalms) [reply images/stable-baselines/Dockerfile:7]: Mmm the docs are confusing, probably it is only needed to build it from sources? I d say to leave it as it is and I will test it with my agent....

File: images/stable-baselines/Dockerfile (hunk 3 above)
Comment (alexpalms) [reply images/stable-baselines/Dockerfile:15]: @discordianfish adding `pip install tensorflow-1.14.0 && \` should be enough, right?

**Comment by discordianfish** [comment]
@alexpalms I've tested sb3 with my agent and it works. Can you give me a sb2 and raylib sample agent? Then I can test this as well.

**Comment by alexpalms** [comment]
> @alexpalms I've tested sb3 with my agent and it works. Can you give me a sb2 and raylib sample agent? Then I can test this as well.

@discordianfish 

### Stable Baselines
You can use the agent we have here: https://github.com/diambra/agents/tree/main/stable_baselines

Making sure that you provide to the pre-build SB image the following (as secrets?):
- agent.py file https://github.com/diambra/agents/blob/main/stable_baselines/agent.py
- cfg file: https://github.com/diambra/agents/blob/main/stable_baselines/cfg_files/doapp/sr6_128x4_das_nc.yaml
- weights file: https://github.com/diambra/agents/blob/main/stable_baselines/results/doapp/sr6_128x4_das_nc/model/25M.zip
- Folder with its content, placing it at the same level of the agent.py: https://github.com/diambra/agents/tree/main/stable_baselines/custom_policies

This should be enough, BUT, I don't see the tensorflow installation instruction in the SB dockerfile (see previous comment in history), I think without it it won t work.

### RLlib

I never trained something, my desktop computer should arrive in a few days and I will restart with my background trainings. The fastest way should be to just run a simple training using one of the examples here https://github.com/diambra/agents/tree/main/ray_rllib and simply dump the model after a single iteration. The creating an agent using it. 

**But I suggest the following: if you can, test SB, I will use RL lib to run my tests about how to use these pre built images** wdyt?

File: images/stable-baselines/Dockerfile (hunk 3 above)
Comment (alexpalms) [reply images/stable-baselines/Dockerfile:15]: @discordianfish I will test it with 1.15.5 even if they say they support up to 1.15.0 (https://stable-baselines.readthedocs.io/en/master/guide/install.html#prerequisites). I will let you know.

**Pull request merged**